### PR TITLE
test: add wasm32 target e2e test

### PR DESCRIPTION
**Pull request by eng-cc**
This Pull Request add a e2e test for wasm32 target.

### CODE REVIEW

File: .github/workflows/webassembly.yml [hunk 1]
```diff
@@ -50,4 +50,8 @@ jobs:
         with:
           node-version: "16"
       - run: npm ci
-      - run: npm run build
+      - name: Install Playwright Browsers
+        run: npx playwright install --with-deps
+      - name: Run Playwright test
+        run: npm run e2e
+      - run: npm run build
```
Comment (Razican) [inline .github/workflows/webassembly.yml:57]: We should add a new line at the end of the file

**Review by Razican — COMMENTED** [review]
Looks pretty good to me! Great addition!

File: .github/workflows/webassembly.yml (hunk 1 above)
Comment (eng-cc) [reply .github/workflows/webassembly.yml]: Thanks for remind, added a new line.

**Comment by eng-cc** [comment]
Wasm32 target is break at main, I use ba7d2163ccad3ce9d5f5ac21893fdefb1cf683ea to test.

**Review by Razican — APPROVED** [review]
This looks good to me :) thank you for the tests!

**Review by jedel1043 — APPROVED** [review]
Thank you for the contribution!

**Comment by eng-cc** [comment]
The old action: `Build webassembly demo` has been deleted and needs to be stopped manually.

**Comment by jedel1043** [comment]
> The old action: `Build webassembly demo` has been deleted,but a job is still created and it is still pending,it needs to be stopped manually.

I checked the full list of pending jobs, but didn't find anything.

**Comment by jedel1043** [comment]
Maybe this is just a Github bug. What I usually do on these cases is to change the base branch to a dummy branch, push a null change then change the base again to main

**Review by HalidOdat — APPROVED** [review]
Looks good to me! Nice Addition! :)

**Comment by eng-cc** [comment]
> Maybe this is just a Github bug. What I usually do on these cases is I change the base branch to a dummy branch, push a null change then change the base again to main

Looks like this didn't work. I try to rebase the main branch and re-trigger the job.

**Comment by jedel1043** [comment]
> Looks like this didn't work. I try to rebase the main branch and re-trigger the job.

Uh I changed the base but couldn't make changes, so I didn't try what I suggested

**Comment by jedel1043** [comment]
The name change fixed it, so I'll push it to the queue. We can try to rename it on a future PR :)

**Pull request merged**